test/compileSync.js

```diff
@@ -0,0 +1,54 @@
+var chai = require("chai")
+var spies = require("chai-spies");
+var path = require("path");
+var compiler = require(path.join(__dirname, ".."));
+
+chai.use(spies);
```

Comment:
Not sure what you are talking about... The requires are all used in the tests. And if you are talking about the tests themselves, them made me find bugs. When I was calling the wrong `spawnFn` for instance. 

Does that answer the question or I missed the point !? Let me know :)